web/src/filter.js

```diff
@@ -222,6 +222,12 @@ export class Filter {
         operator = Filter.canonicalize_operator(operator);
 
         switch (operator) {
+            case "is":
+                // deprecated "is:private" narrow
```

Comment:
I would write "// is:private was renamed to is:dm" here -- that concisely explains the concrete reason why this is correct.

Reply:
Done.